dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifiableRepositoryImpl.java

```diff
@@ -496,22 +447,21 @@ public PageResponse<I> findByLanguageAndInitial(
     }
     pageRequest.setSorting(sorting);
 
-    Map<String, Object> argumentMappings = new HashMap<>();
+    Map<String, Object> argumentMappings = new HashMap<>(0);
```

Comment:
```suggestion
    Map<String, Object> argumentMappings = new HashMap<>(2);
```
Die folgenden zwei Zeilen adden ja gleich 2 Werte.